code/ui/manager/src/components/sidebar/TreeNode.tsx

```diff
@@ -145,6 +142,26 @@ export const RootNode = styled.div(({ theme }) => ({
   color: theme.textMutedColor,
 }));
 
+const Wrapper = styled.div({
+  display: 'flex',
+  alignItems: 'center',
+});
+
+const InvisibleText = styled.p({
+  margin: 0,
+  width: 0,
+});
+
+// Make the content have a min-height equal to one line of text
+export const IconsWrapper: FunctionComponent<{ children?: React.ReactNode }> = ({ children }) => {
```

Comment:
You can use `FC` instead of `FunctionComponent`

Reply:
Good to know, this is better 😄 
I changed it here: https://github.com/storybookjs/storybook/pull/22221/commits/f5621205dc714499563f9cb9061f421756a1d632